contents/docs/web-analytics/web-vs-product-analytics.mdx

```diff
@@ -0,0 +1,225 @@
+---
+title: What is the difference between web analytics and product analytics?
+sidebar: Docs
+showTitle: true
+---
+
+import { ComparisonTable } from 'components/ComparisonTable'
+import { ComparisonRow } from 'components/ComparisonTable/row'
+
+- [Web analytics](/web-analytics) is for tracking website metrics. It's for marketing websites, blogs, or high volume B2C apps where users aren't logged in.
+- [Product analytics](/product-analytics) is for tracking your product. It's more powerful than web analytics. It's for web and mobile apps.
```

Comment:
These intro bit doesn't quite land for me atm. Couple of things:

- I think it's better to say "they're good for X" instead "they are for X" – i.e. "web analytics is good for when you want a simplified view etc", "product analytics is good for diving deep and building custom insights".

- We shouldn't say web analytics is for B2C apps, imo. You can use it for that, but it's confusing to say it's web analytics and then talk about apps instead of websites, and these users should probably use product analytics a lot of the time.

- I don't love vague terms like "most powerful". This doesn't mean much.

The TL;DR: we need to explain the difference without creating a bunch of rules around how people use them, if that makes sense?

Reply:
Agreed. I ended up changing the intro a handful of times and it definitely lost some context and clarity after tightening it up. I ended up doing a small rewrite to incorporate the feedback above:
https://posthog-git-docs-web-vs-product-analytics-ii-post-hog.vercel.app/docs/web-analytics/web-vs-product-analytics